.travis.yml

```diff
@@ -150,14 +150,34 @@ install:
   - cd ..
   - git clone -b master --depth 1 https://github.com/boostorg/boost.git boost-root
   - cd boost-root
+  # List deps explicitly to control any new additions
   - git submodule update --init tools/build
-  - git submodule update --init libs/config
   - git submodule update --init tools/boostdep
+  - git submodule update --init libs/algorithm
+  - git submodule update --init libs/array
+  - git submodule update --init libs/bind
+  - git submodule update --init libs/concept_check
+  - git submodule update --init libs/config
+  - git submodule update --init libs/core
+  - git submodule update --init libs/crc
+  - git submodule update --init libs/filesystem
+  - git submodule update --init libs/function
+  - git submodule update --init libs/integer
+  - git submodule update --init libs/iterator
+  - git submodule update --init libs/lambda
+  - git submodule update --init libs/lexical_cast
+  - git submodule update --init libs/mpl
+  - git submodule update --init libs/numeric/conversion
+  - git submodule update --init libs/preprocessor
+  - git submodule update --init libs/smart_ptr
+  - git submodule update --init libs/static_assert
+  - git submodule update --init libs/test
+  - git submodule update --init libs/type_traits
+  - rm -rf libs/gil
+  - mkdir libs/gil
   - cp -r $TRAVIS_BUILD_DIR/* libs/gil
-  - cp -r $TRAVIS_BUILD_DIR/.ci libs/gil
-  - python tools/boostdep/depinst/depinst.py gil
   - ./bootstrap.sh
-  - ./b2 headers
+  # - ./b2 headers REQUIRED?
```

Comment:
With `b2` that's implicit, but I actually prefer it to be explicit (for similar reasons as your addition above), and it may also be needed for other build systems (cmake, faber), unless we handle include search paths explicitly there (which wouldn't be a bad thing either !)

Reply:
Good. I've restored it.